content/v2.0/reference/flux/stdlib/experimental/geo/_index.md

```diff
@@ -0,0 +1,118 @@
+---
+title: Flux Geo package
+list_title: Geo package
+description: >
+  The Flux Geo package provides tools for working with geo-temporal data and performing
+  geographic location filtering and grouping.
+  Import the `experimental/geo` package.
+menu:
+  v2_0_ref:
+    name: Geo
+    parent: Experimental
+weight: 201
+v2.0/tags: [functions, package, geo]
+---
+
+The Flux Geo package provides tools for working with geo-temporal data and performing
```

Comment:
same commend as above. (if filtering and grouping geographic location isn't a subset of working w geo-temporal data, we could keep "and" in there:
...geo-temporal data and filtering and grouping geographic location.